core-it-suite/src/test/resources/mng-6720-fail-fast/pom.xml

```diff
@@ -35,15 +35,4 @@ under the License.
     <module>module-3</module>
   </modules>
 
-  <build>
-    <pluginManagement>
-      <plugins>
-        <plugin>
-          <groupId>org.apache.maven.plugins</groupId>
-          <artifactId>maven-resources-plugin</artifactId>
-          <version>3.2.0</version>
-        </plugin>
-      </plugins>
-    </pluginManagement>
-  </build>
```

Comment:
Why not use plugin management in the part for those?

Reply:
because project use aggregation and child poms don't have defined parent - it is main reason for change

Reply:
Ah, yes. Make sense.